examples/quickstart/run.py

```diff
@@ -53,14 +51,12 @@
     help="Choose the model size: t5-small or t5-large.",
 )
 @click.option(
-    "--orchestration_environment",
-    type=click.Choice(["local", "aws", "gcp", "azure"], case_sensitive=False),
-    default="local",
-    help="Choose the orchestration environment.",
+    "--config_path",
+    help="Choose the configuration file.",
 )
 def main(
     model_type: str,
-    orchestration_environment: str,
+    config_path: str,
```

Comment:
If this argument is optional, shouldn't this be `Optional[str] = None` instead ?